source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -481,6 +486,109 @@ ShaderGraphPtr ShaderGraph::create(const ShaderGraph* parent, const string& name
         // Start traversal from this output
         root = output;
     }
+    else if (element->isA<Node>())
```

Comment:
New support for generating shaders from Node elements. In this mode only the single node is considered and any upstream dependencies are ignored.